src/platform.ts

```diff
@@ -107,52 +120,48 @@ function getRuntimeIdLinux(distributionName: string, distributionVersion: string
     switch (distributionName) {
         case 'ubuntu':
             if (distributionVersion.startsWith('14')) {
-                // This also works for Linux Mint
                 return Runtime.Ubuntu_14;
             } else if (distributionVersion.startsWith('16')) {
                 return Runtime.Ubuntu_16;
+            } else if (distributionVersion.startsWith('18')) {
+                return Runtime.Ubuntu_18;
+            } else if (distributionVersion.startsWith('20')) {
+                return Runtime.Ubuntu_20;
+            } else if (distributionVersion.startsWith('22')) {
+                return Runtime.Ubuntu_22;
+            } else {
+                return Runtime.Ubuntu;

```

Comment:
Might be a good idea to log a warning too if it's an unknown version.

Reply:
done